app/controllers/students_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
 
   def show
     @student = Student.find params[:id]
+    @group = Group.where id: @student.group_id
```

Comment:
No need to do this! You can get it with `@student.group`, if group could be null use `@student.group.try(:name)` yay bonads!